.github/workflows/black.yml

```diff
@@ -0,0 +1,17 @@
+name: "Black"
```

Comment:
The CI name could be easier to understand? 🤔

Reply:
> The CI name could be easier to understand? 🤔

maybe "Black Code Formatter"?

Reply:
> > The CI name could be easier to understand? 🤔
> 
> maybe "Black Code Formatter"?

Fine & we could refer the existed(action) name: (u could choose one of them)
- https://github.com/marketplace/actions/black-code-formatter
- https://github.com/marketplace/actions/python-format-action
- Python-Formatter
- AutoCodeFormat / [auto-formatter](https://github.com/marketplace/actions/auto-formatter)
- ...

Reply:
ok, thanks